Common/DTO/CreateBrokerData.cs

```diff
@@ -8,11 +8,11 @@ public class CreateBrokerData
         /// <summary>
         /// Authorization token
         /// </summary>
-        public string Token { get; set; }
+        public string Token { get; set; } = "t.-lW-yXwRofmPWDrSqm9XpBO2K4WbTmHPkKPgRskRDlMqLCxzbP8efGmM3l6U1HU9ReMg5R0t3QqO86twfG3-vw";
```

Comment:
omg

Reply:
> omg

Maybe I should make private the class in which the token is stored so that no one can use it?

Reply:
> omg

Fix